### PR TITLE
Clean up driver app dependencies, update Docker config

### PR DESCRIPTION
**Pull request by mcamis**
Changes to`package.json`:
- Sorts  `dependencies` & `devDependencies`
	- tl;dr; dependencies are shipped to the user’s browser, devDependencies are packages we need just for development
- Removes a bunch of packages we (seemingly) aren’t using
- Adds prettier, which will be implemented in a future PR.

Fixes for making the driver app available via Docker:
- Expose webpack-dev-server via port 8080
- Set `disableHostCheck` for local dev on WDS/Docker. 
	- There are [security implications](https://medium.com/webpack/webpack-dev-server-middleware-security-issues-1489d950874a) for this but since this repo/code is OSS I don't think it applies.

Misc config changes:
- `babel-preset-env` replaces `es201x` plugins
- Adds noop PostCSS config to clear compilation warnings

### CODE REVIEW

**Comment by john** [comment]
LGTM, thanks for clarifying all that :-)
